lib/money-rails/active_record/monetizable.rb

```diff
@@ -85,10 +85,10 @@ def monetize(*fields)
             #
             # To disable validation entirely, use :disable_validation, E.g:
             #   monetize :price_in_a_range_cents, :disable_validation => true
-            if validation_enabled = MoneyRails.include_validations && !options[:disable_validation]
+            if (validation_enabled = MoneyRails.include_validations && !options[:disable_validation])
```

Comment:
Hmm… not sure about this one. Is it because of the assignment inside a condition?